data/net/minecraft/util/ExtraCodecs.mapping

```diff
@@ -87,26 +87,62 @@ CLASS net/minecraft/util/ExtraCodecs
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 other
 	CLASS EitherCodec
+		COMMENT A codec implementation for handling the Either data type, which represents a value that can be either of two alternative types, F or S.
+		COMMENT <p>
+		COMMENT This codec combines two existing codecs for the two alternatives and handles decoding and encoding based on the actual data type encountered during serialization and deserialization.
```

Comment:
This is news to me that Mojang ships an `EitherCodec` that literally replaces DFU's `EitherCodec`, with one that has better error handling, instead of ya know, fixing it in DFU. Mojang fix your upstream sheesh.

In other news, I think this documentation should focus less on what this codec does, and more on "This is `Codec.either` but with a key change (in the case that both decodings return an error, returns `apply2()` instead of the second error. Because in all other respects it's identical.